pymc/tests/test_sampling.py

```diff
@@ -1134,11 +1102,18 @@ def test_multivariate2(self):
             return_inferencedata=False, samples=20, model=dm_model
         )
         sim_ppc = pm.sample_posterior_predictive(
-            burned_trace, return_inferencedata=False, samples=20, model=dm_model
+            burned_trace, return_inferencedata=False, model=dm_model
         )
         assert sim_priors["probs"].shape == (20, 6)
         assert sim_priors["obs"].shape == (20,) + mn_data.shape
-        assert sim_ppc["obs"].shape == (20,) + mn_data.shape
+        assert (
+            sim_ppc["obs"].shape
+            == (
+                1,
+                20,
```

Comment:
If you remove the comma, black will not break this across multiple lines

```suggestion
                20
```